index.html

```diff
@@ -92,100 +171,184 @@ <h6>Skills</h6>
+                <div class="card-one">
+                    <img class="card-image" src="./ImgPlaceholder.png" alt="" />
+                    <div class="stoies">
+                        <h2 class="post-heading">Multi-Post Stories</h2>
+                        <p class="post-1">
+                            A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. has been the industry's standard dummy text ever
+                            since the 1500s, when an unknown printer took a standard dummy text.
+                        </p>
+                        <ul class="languages">
+                            <li>css</li>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>ruby</li>
+                        </ul>
+                        <button class="btn" type="button">See Project</button>
+                    </div>
+                </div>
+                <div class="card-two">
+                    <img class="card-image" src="./ImgPlaceholder.png" alt="" />
+                    <div class="stoies" id="end">
+                        <h2 class="post-heading">Multi-Post Stories</h2>
+                        <p class="post-1">
+                            A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. has been the industry's standard dummy text ever
+                            since the 1500s, when an unknown printer took a standard dummy text.
+                        </p>
+                        <ul class="languages">
+                            <li>ruby</li>
+                            <li>bootstrap</li>
+                            <li>html</li>
+                            <li>css</li>
+                        </ul>
+                        <button class="btn" type="button">See Project</button>
+
+                    </div>
+
+                </div>
+                <div class="card-one">
+                    <img class="card-image" src="./ImgPlaceholder.png" alt="" />
+                    <div class="stoies">
+                        <h2 class="post-heading">Multi-Post Stories</h2>
+                        <p class="post-1">
+                            A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. has been the industry's standard dummy text ever
+                            since the 1500s, when an unknown printer took a standard dummy text.
+                        </p>
+                        <ul class="languages">
+                            <li>css</li>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>ruby</li>
+                        </ul>
+                        <button class="btn" type="button">See Project</button>
+                    </div>
+                </div>
```

Comment:
- [ ] For every project, align the image and button to the bottom. Compare your page Vs the original.
<div class="flex">
<img src="https://user-images.githubusercontent.com/65501535/134750371-5667d46f-4225-430e-bef7-a33da7853c58.png"  style="width: 50%;"/>
<img src="https://user-images.githubusercontent.com/65501535/134750319-7ff756f1-4e12-4c14-9e7a-0709868abe11.png"  style="width: 50%;"/>
</div>